src/tox/tox_env/runner.py

```diff
@@ -143,6 +143,10 @@ def _setup_with_env(self) -> None:
             else:
                 self._setup_pkg()
 
+    def _done_with_setup(self) -> None:
```

Comment:
Why do we need this, considering https://github.com/tox-dev/tox/blob/main/src/tox/tox_env/python/api.py#L216. Shouldn't we be reusing that call?

Reply:
This is python specific so I don't see why it would leave under the generic runner and not the python scope.

Reply:
> So in the CI we can pass this `--display-dependencies` argument and locally it's also possible to pass this argument if the user has any reason to do it, but by default it will not change anything.

I'd argue in the CI envs we should do this without the user passing any flags. And that was my threshold of acceptability in the issue.